Logger/FileLogger.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+
+namespace Logger
+{
+    public class FileLogger : BaseLogger
+    {
+        private string filePath;
```

Comment:
Fields should be underscore PascalCase
```suggestion
        private string _FilePath;
```